File: templates/matlab/bio_signal_stage_step.m
function fig = bio_signal_stage_step()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('step_curve', 2717, 'biomedical signal analysis: stage step curve', 'biomedical signal analysis', 'stage step curve');
end

File: templates/python/bio_signal_stage_step.py
"""bio_signal_stage_step: 生物信号阶段阶梯曲线（step 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='biomedical signal analysis: stage step curve'):
    return make_template_figure('step_curve', seed=2717, title=title, domain='biomedical signal analysis', topic='stage step curve')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
